WORKSPACE

```diff
@@ -574,16 +572,25 @@ hazel_repositories(
                 "a2be2d812010eaadd4885fb0228370a5467627bbb6bd43177fd1f6e5a7eb05f8",
                 patch_args = ["-p1"],
                 patches = ["@com_github_digital_asset_daml//bazel_tools:haskell-lsp-test-no-reexport.patch"],
-            ) +
-            hazel_github_external(
+            ) + hazel_github_external(
                 "mpickering",
                 "hie-bios",
                 "8427e424a83c2f3d60bdd26c02478c00d2189a73",
                 "c593ff871f31200e37a3c24c09da314d0ee41a8486defe7af91ac55a26efdc1e",
                 patch_args = ["-p1"],
                 patches = ["@com_github_digital_asset_daml//bazel_tools:haskell-hie-bios.patch"],
-            ) +
-            hazel_hackage("typed-process", "0.2.6.0", "31a2a81f33463fedc33cc519ad5b9679787e648fe2ec7efcdebd7d54bdbbc2b1"),
+            ) + hazel_hackage("typed-process", "0.2.6.0", "31a2a81f33463fedc33cc519ad5b9679787e648fe2ec7efcdebd7d54bdbbc2b1") + [
+                (
+                    "c2hs",
```

Comment:
`hazel_hackage` can also take a `patches` argument. See e.g. `network` above.

Reply:
Good point, thanks! I changed it.